### PR TITLE
chore(README): Where to put the files + v1

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,23 @@ Please see [action.yml](action.yml)
 
 ## Example Usage
 
-Place this file in ".github/workflows/action.yml". 
+You need two things: 
+ 1. an "ECHIDA" file
```
Comment (sidvishnoi) [inline README.md]:
```suggestion
 1. an "ECHIDNA" file
```

Comment (marcoscaceres) [reply README.md]: Um! That was test, just to see if you were paying attention... you passed.

File: README.md [hunk 2]
```diff
@@ -13,6 +13,24 @@ Please see [action.yml](action.yml)
...
+
+Just change `your-spec-shortname` to whatever your spec's short name is. 
+
+```
+index.html?specStatus=WD&shortName=your-spec-shortname respec
+```
+
+### action.yml
+
+Place this file in ".github/workflows/action.yml" and change things as appropriate for your spec.
```
Comment (sidvishnoi) [inline README.md]: Maybe also let know the file name doesn't necessarily need to be `action.yml`.

File: README.md [hunk 3]
```diff
@@ -28,10 +45,11 @@ jobs:
     runs-on: ubuntu-latest # only linux supported at present
     steps:
       - uses: actions/checkout@v1
-      - uses: w3c/respec-w3c-auto-publish@master # use the action
+      - uses: w3c/respec-w3c-auto-publish@v1 # use the action
         with:
           ECHIDNA_TOKEN: ${{ secrets.ECHIDNA_TOKEN }}
-          ECHIDNA_MANIFEST_URL: "https://w3c.github.io/gamepad/W3CTRMANIFEST"
+          ECHIDNA_MANIFEST_URL: "https://w3c.github.io/your_spec_repo/ECHIDNA"
+          # Please use the URL that's appropriate for your working group!
```
Comment (marcoscaceres) [inline README.md:52]: We should hunt these down... there is probably only like 5-10 URLs.

File: README.md [hunk 4]
```diff
@@ -13,6 +13,23 @@ Please see [action.yml](action.yml)
 
 ## Example Usage
 
+You need two things: 
+ 1. an "ECHIDNA" file
```
Comment (sidvishnoi) [inline README.md]:
```suggestion
 1. an "ECHIDNA" manifest file
```
Echidna calls it manifest file: https://github.com/w3c/echidna/wiki/Preparing-your-document#manifest-file

**Review by sidvishnoi — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (sidvishnoi) [reply README.md:52]: Filed https://github.com/w3c/respec-w3c-auto-publish/issues/12

**Pull request merged**